dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/sms/listener/AggregateDataSetSMSListener.java

```diff
@@ -110,10 +110,10 @@ protected SMSResponse postProcess( IncomingSms sms, SMSSubmission submission )
         String per = subm.getPeriod();
         UID aocid = subm.getAttributeOptionCombo();
 
-        OrganisationUnit orgUnit = organisationUnitService.getOrganisationUnit( ouid.uid );
-        User user = userService.getUser( subm.getUserID().uid );
+        OrganisationUnit orgUnit = organisationUnitService.getOrganisationUnit( ouid.getUID() );
+        User user = userService.getUser( subm.getUserID().getUID() );
```

Comment:
I hate to harp on this, but we use `Uid` instead of `UID` in DHIS 2. Using camel case even for acronyms is a standard Java convention. Would you mind changing it to `getUid()` ?

Reply:
@larshelge - hmmm, the Java API is anything but consistent on this... :)

But I can see it's certainly a convention in DHIS2 core code. I've gone and changed all acronyms in the SMS compression library now.

I've got 3 PRs open at the moment, for 2.33, 2.34 and master. Is it worth updating all with this? Or maybe I can just add the change to master?

Reply:
Also I've just noticed a lot of the existing SMS listener code already used SMS in its class names. I suppose we should be consistent and update all these too?

Reply:
Okay great, thanks @JasperTimm. To avoid a growing task let's leave it at Uids for now. 

It would be great if you could update the 2.33 and 2.34 PRs as well, for consistency and for easier future backports.

Reply:
@larshelge - case of acronym PR here: https://github.com/dhis2/dhis2-core/pull/5422